src/Eras.Api/Controllers/EvaluationController.cs

```diff
@@ -48,5 +46,16 @@ public async Task<IActionResult> CreateEvaluation([FromBody] EvaluationDTO evalu
                 );
             }
         }
+
+        [HttpGet("summary")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> GetEvaluationProcessSumary()
```

Comment:
just a little typo here
```suggestion
        public async Task<IActionResult> GetEvaluationProcessSummary()
```